assume/strategies/naive_strategies.py

```diff
@@ -76,7 +76,7 @@ def calculate_bids(
         return bids
 
 
-class NaiveDAStrategy(BaseStrategy):
```

Comment:
I think this is much better renamed to NaiveProfileStrategy? I find it weird to tie the name to the DAM?
But if you think it is better like this, it is fine for me.

Same for the NaiveEOM -> NaiveSingleBidStrategy?

Reply:
yeah true, will change it